lib/rules/a11y-svg-has-accessible-name.js

```diff
@@ -0,0 +1,41 @@
+const {hasProp} = require('jsx-ast-utils')
+const {getElementType} = require('../utils/get-element-type')
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'SVGs must have an accessible name',
+      url: require('../url')(module),
+    },
+    schema: [],
+  },
+
+  create(context) {
+    return {
+      JSXOpeningElement: node => {
+        const elementType = getElementType(context, node)
```

Comment:
An early return here will speed up execution which can get bogged down with many small cuts:

```suggestion
        const elementType = getElementType(context, node)
        if (elementType !== 'svg') return
```